### DISCUSSION TITLE
wiki/overview/faq/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#649] [Wiki]
# wiki/overview/faq/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/overview/faq/

**Comment by matvii606** [comment]
why when i go to modrinth launcher i try to log in and i get a 403 error and it says i didn't buy minecraft but when i log in with the same account and prism launcher i get what should I do to enter modrinth launcher?

**Reply by Trial97** [reply]
You should ask this on modrinth side, not Prism.

**Reply by matvii606** [reply]
I was told in discord here

**Reply by matvii606** [reply]
I was told in discord here

**Reply by Trial97** [reply]
Prism follows https://wiki.vg/Microsoft_Authentication_Scheme 
You should ask this on the modrinth discord as there is not much a Prism maintainer can help.

**Comment by matvii606** [comment]
why when i go to modrinth launcher i try to log in and i get a 403 error and it says i didn't buy minecraft but when i log in with the same account and prism launcher i get what should I do to enter modrinth launcher?

**Comment by hannahchinyere** [comment]
Could someone please help me. I feel like i’ve tried everything. (trying different versions, deleting and reinstalling prism) I have a steam deck and minecraft works on desktop mode. However when in gaming mode it either doesn’t start, gets frozen when loading mojang (using version 1.20.1 w fabric) and with forge it does load, until an error saying: ‘failed to initialize graphics window with current settings’.
At this point, idk what to do. I looked at tutorials as well for the steam deck but everyone can launch it in gamingmode without issues :(.

**Comment by Lamby777** [comment]
came here for a link to that specific commit for reference, but i just realized there's a flatpak and i can probably install this on steam deck :o

**Comment by ZaharitIngot** [comment]
what the heaul is QT and how i find it
